exampleSite/config.toml

```diff
@@ -25,6 +25,9 @@ disqusShortname = "yourdiscussshortname"
     hideCredits = false
     hideCopyright = false
 
+    # Git Commit in Footer, uncomment the line below to enable it.
+    commit = "https://github.com/luizdepra/hugo-coder/tree/"
```

Comment:
Hmm... Maybe we should follow the idea of the another footer flag here. Maybe you should create a `hideCommit`flag and get the the full repository address from a env var too.

Reply:
I actually wanted to open a PR for this, separately, to move footer related configs to `[params.footer]`.
Although now I'm thinking the missing feature in this PR is the ability to configure a text for the commit. something like `This site is built from` which can be configured.

Reply:
Let me know what you guys decide, and I can make the changes.  I am real busy this week though, so it may be a bit before I can get to it.

I have seen the git commit in the footer on a couple themes, none of them used an env. variable for the address though.

Reply:
@luizdepra Found a better way to handle the env vars?

Reply:
Sorry, I did not got the time to do it. We can leave it that way and change later, no problem.